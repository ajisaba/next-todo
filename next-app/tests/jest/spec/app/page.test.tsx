import HomePage from "@/app/page";
import { render } from "@testing-library/react";

describe("テスト・HomePage", () => {
  test("初期表示", async () => {
    const { asFragment, getByRole } = render(<HomePage />);

    expect(getByRole("heading", { level: 1 }).innerHTML).toBe("Next-Todo");

    expect(asFragment()).toMatchSnapshot();
  });
});
