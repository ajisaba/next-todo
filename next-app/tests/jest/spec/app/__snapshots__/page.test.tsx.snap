// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`テスト・HomePage 初期表示 1`] = `
<DocumentFragment>
  <main
    class="w-full text-center p-4"
  >
    <h1
      class="text-xl"
    >
      Next-Todo
    </h1>
    <div
      class="mt-4"
    >
      <ul>
        <li>
          <a
            class="underline"
            href="/schedule"
          >
            予定
          </a>
        </li>
      </ul>
    </div>
  </main>
</DocumentFragment>
`;
